tests/smoke/test_cli.py

```diff
@@ -41,3 +42,11 @@ def test_cli_reg_metrics_runs_with_output(capfd, tmpdir):
     os.system(f'reg_metrics data/test_data/ --tiff_list input_1500.tif --do_registration 2 --save_path0 {tmpdir}')
     captured = capfd.readouterr()
     assert captured.out
+
+
+def test_cli_suite2p_gui_runs_when_is_called_locally():
+    s2p = subprocess.Popen(['python', '-m', 'suite2p'], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    sleep(10)
+    s2p.terminate()
```

Comment:
The test seems to happen only after the application has been terminated.  It would be great if we could test that the gui is actually running before the terminate() call happens.

Reply:
Yep, it runs after it terminates because I wanted to mainly check that the `suite2p` command didn't result in any errors.